### PR TITLE
Interactive lists

### PR DESCRIPTION
**Pull request by damdafayton**
## Project requirements

- Add a new JavaScript file and import it as a module: it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/modules/checkBoxControl.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      index = i;
+    }
+  });
+  // find task within localstorage
+  const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  tasks.forEach((task) => {
+    console.log(index);
+    if (task.index === index) {
+      task.completed = completeStatus;
+      console.log(task);
```
Comment (micheaol) [inline src/modules/checkBoxControl.js]: Please ensure that you do not have console.log(task); in your codebase, it makes your code clean and for security reasons.

File: src/modules/populateList.js [hunk 2]
```diff
@@ -1,45 +1,8 @@
...
-      index = i;
-    }
-  });
-  // find task within localstorage
-  const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
-  tasks.forEach((task) => {
-    console.log(index);
-    if (task.index === index) {
-      task.completed = completeStatus;
-      console.log(task);
```
Comment (micheaol) [inline src/modules/populateList.js:26]: Please ensure that you do not have console.log(task); in your codebase, it makes your code clean and for security reasons.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @damdafayton , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @damdafayton,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**